### ISSUE TITLE
Quest - write the MVP of new ember-cli-guides

### ISSUE THREAD

**Issue by jenweber** [ember-learn/cli-guides#3]
Thanks for your interest in helping with this project! We are porting over content from [ember-cli.com](https://ember-cli.com), editing it, and writing new things. This project is part of an [RFC](https://github.com/ember-cli/rfcs/pull/120)

Choose a task, comment on this issue to say which section, then open an issue with the same name. The newly opened issue is the best place for Q&A about the task. Please read [CONTRIBUTING.md](https://github.com/ember-learn/cli-guides-source/blob/master/CONTRIBUTING.md) before starting to write. Note that you should not have to create any new files. If you need help deciding where your content belongs, ask a question on the issue and @ me.

We will accept PRs that partially address topics, add comments with suggested content, or start to define subheadings within topics. Small PRs are awesome! The goal is to be quick to PR, quick to merge, and allow for some churn later on.

**Note on respecting copyright** - It is ok to copy/paste content over from ember-cli.com. It is _not_ ok to copy and paste content that you didn't write that came from blog posts, articles, tutorials, etc. whether paid or free. Exceptions are if the license allows it or you have written permission of the author via a comment on an issue in this repo.

Goal -  Ship by the end of November!

## Open tasks that need volunteers:
- [x] Pick a markdown file without content and add suggested headings (many people needed, name a subject and it's yours. Make it clear in your comment or issue name.)
- [x] Update Asset Compilation
- [x] Update Upgrading section (separate PR)
- [x] Port over up-to-date sections of [Blueprints](https://ember-cli.com/extending/#generators-and-blueprints). Remove/edit outdated content. Recommended for experienced addon authors.
- [x] Go through [developing addons](https://ember-cli.com/extending/#developing-addons-and-blueprints) and open an issue listing points missing from the Tutorial
    - [ ] After the audit, PR the additions (should be very small, 5 sentences max)
- [x] Write a section on including assets in addons - Greenfield, recommended for experienced writer/addon author.
- [ ] Write updating addons section in the "Upgrading" page

## WIP tasks

- [x] Port over [Environments](https://ember-cli.com/user-guide/#Environments). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @ddoria921

## On hold (no volunteers needed yet)
- [ ] create a mapping of ember-cli.com pages to our urls
- [x] End to end prose editing sweep after MVP is reached
- [ ] End to end technical review
- [x] CI
- [x] Staging deployment
- [x] clean up some stylesheet quirks
- [x] handle problems with incomplete urls
- [x] alpha deployment to cli.emberjs.com
- [ ] implement redirects in ember-cli.com (after Jan 1st?)
- [x] add cli team as maintainers for the github repo

## Completed tasks
- [x] align RFC topics with the table of contents/file structure - @jenweber 
- [x] write basic beginner content for using the CLI and making an addon - @jenweber 
- [x] app structure, deploy pipeline, scaffolding, etc - thank you so much @mansona  
- [x] Port over [watchman instructions](https://ember-cli.com/user-guide/#watchman). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) #4 @sandstrom, includes review and updates
- [x] Port over Windows-specific instructions on [ember-cli.com](ember-cli.com). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @Gaurav0  #6 
- [x] Port over [Editors](https://ember-cli.com/user-guide/#editors). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @cah-danmonroe  #6 
- [x] Automated testing by @aklkv #9 
- [x] Port over [Asset Compilation](https://ember-cli.com/user-guide/#asset-compilation) (unedited). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @Gaurav0 #18 
- [x] Port over [Common Issues](https://ember-cli.com/user-guide/#commonissues). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @BradenLawrence #12 
- [x] Port over [Upgrading](https://ember-cli.com/user-guide/#upgrading). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @DanMonroe #11 
- [x] Port over [managing dependencies](https://ember-cli.com/user-guide/#managing-dependencies). Mostly copy and paste from [this source code](https://github.com/ember-cli/ember-cli.github.io) @BradenLawrence #19 
- [x] Update or rewrite Managing Dependencies (a separate PR from above) @jenweber 
- [x] Rewrite [Deployments](https://ember-cli.com/user-guide/#deployments) - greenfield, recommended for experienced writers @jenweber

**Comment by jenweber** [comment]
## Completed tasks:

**Comment by sandstrom** [comment]
I've made a PR for **Port over watchman instructions**: https://github.com/ember-learn/cli-guides-source/pull/4

**Comment by cah-danmonroe** [comment]
I'll take a look at porting Editors

**Comment by Gaurav0** [comment]
I've made a PR for port over Windows support (#6).

**Comment by BradenLawrence** [comment]
I'll start working on:
"Port over Common Issues. Mostly copy and paste from this source code"

**Comment by cah-danmonroe** [comment]
I'll do Port over Upgrading.

**Comment by ddoria921** [comment]
I'll work on:
"Port over Environments. Mostly copy and paste from this source code"

**Comment by BradenLawrence** [comment]
Taking this one next:
"Port over Environments. Mostly copy and paste from this source code"

**Comment by ddoria921** [comment]
@BradenLawrence I started on that one yesterday 😄

**Comment by BradenLawrence** [comment]
So you did! Sorry I missed that. Taking this one instead:
"Port over managing dependencies. Mostly copy and paste from this source code"

**Comment by maxwondercorn** [comment]
I'm starting to work on blueprints. I maintain a few addons, but someone with more experience than may need to flush it out after I'm done

**Comment by jenweber** [comment]
Good news, everyone! We have reached MVP and the launch is scheduled for Monday, November 26th! Tomorrow, the content will become available at https://cli.emberjs.com.  Thank you so much for all your work ❤️

The next phase for this project will focus on resources for addon developers. We will also implement redirects from the old CLI site, and add links to this site from many places. If you would like to help, keep an eye out for new issues on this project.

Closed with release of [1.0](https://github.com/ember-learn/cli-guides/releases/tag/1.0.0)
